.github/workflows/test.yaml

```diff
@@ -1,5 +1,12 @@
 name: test
-on: push
+on:
+  push:
+    branches:
+      - master
+      - alpha
+      - dev
+      - patch-dev
+  pull_request:
```

Comment:
is this correct with the colon but not sub value?

Reply:
yep

Reply:
seems a bit weird, but works indeed